nats-server/channelsz.yml

```diff
@@ -0,0 +1,40 @@
+type: json
+data:
+  url: /streaming/channelsz
+  server: ${NATS_SERVER_HOSTPORT}
+
+observation:
+  - name: channelsz
+    metricNamespace: nats
+    path: $.
+
+    metric:
+      - name: streaming.channels.total
+        source: total
+        type: long_gauge
+        label: total channels
+
+      - name: streaming.channels.limit
+        source: limit
+        type: long_gauge
+        label: max channels
+
+  - name: channelsz_channel
+    metricNamespace: nats
+    path: $.channels[?(@.name=${channelName}})]
+
+    metric:
+      - name: streaming.channels.messages
+        source: msgs
+        type: counter
+        label: The total of messages
+
+      - name: streaming.channels.bytes
+        source: bytes
+        type: counter
+        unit: bytes
+        label: The total of bytes
```

Comment:
Can we rename the label? `The total of bytes` seems very vague. maybe `bytes streamed`? Also, remove `The` from labels. We don't use it for labels.

Reply:
Rename it and some similar metric.